src/main/resources/reference.conf

```
@@ -722,3 +722,5 @@ distributedLock {
   retryInterval: 5s
   maxRetry: 10
 }
+
+domainBlacklist = ["qwiklabs-gsuite.net", "qwiklabs.net"]
```

I'm torn as to whether this information should be in `reference.conf` or `sam.conf`.  I always just thought of `reference.conf` as kind of `resource_types.conf`.  I see now that there are some other settings in there, so I think the question is, what are the guidelines for determining which settings go into `reference.conf` and which go into `sam.conf`?

---

```suggestion
domainBlocklist = ["qwiklabs-gsuite.net", "qwiklabs.net"]
```
And replace all variables as well, `blacklist` -> `blocklist`

---

new term: blockedEmailDomains